content/core-features/authentication.textile

```diff
@@ -211,6 +211,53 @@ ably.auth.requestToken({ clientId: 'client@example.com' }, function(err, tokenDe
 
 *Note:* The authenticity of the JWT *will not* be checked, due to Ably not having access to your SECRET key.
 
+h4(#jwt-claims). Using JWT for authenticated user claims
+
+It is possible to use JWTs to authenticate claims for users in your channels, allowing or disallowing certain interactions.
+
+Messages can include trusted fields in the `extras` object that contain information copied from their authentication token, making it unique to each user. These fields are deemed trusted as their content is added is supplied directly from Ably and not the client. The trusted field is a channel-specific piece of information and needs to be set for other channels if similar permissions are desired.
+
+*Note:* This does not apply to presence or metadata messages.
+
+An example use case is when using "Message Interactions":/realtime/messages#message-interactions you might want to create a moderator in a chat channel that has the ability to delete sent messages. When the moderator sends an interaction request to delete a message Ably checks that the user's claim for a value moderator for that channel before actioning their request.
+
+To set the trusted fields you need to include `ably.channel.` in your JWT authentication payload, for example:
+
+```[javascript]
+const claims = {
+  "sub": "1234567890",
+  "name": "John Doe",
+  "x-ably-capabilities": <...>,
+  "x-ably-clientId": <...>,
+  "ably.channel.chat1":  <opaque data>, // claims for just the chat1 channel
+  "ably.channel.chat:*": <opaque data>, // claims for channels in the "chat" namespace
+  "ably.channel.*":  	<opaque data>, // claims for all channels
+}
+```
+
+This is an example of how you might check if a message has permission:
```

Comment:
This is an example of how you might check if a message has permission. Note how the claims from the token are copied onto messages, so that they may be checked.